stan/math/rev/core/var.hpp

```diff
@@ -320,11 +235,12 @@ class var {
    * (a += b) behaves exactly the same way as (a = a + b).  Note
    * that the result is an assignable lvalue.
    *
+   * @tparam Arith An arithmetic type
    * @param b The scalar to add to this variable.
    * @return The result of adding the specified variable to this variable.
    */
-  template <typename Arith, require_arithmetic_t<Arith>...>
-  inline var& operator+=(Arith b);
+  template <typename Arith, require_arithmetic_t<Arith>* = nullptr>
```

Comment:
Shouldn't this be value_type now instead of arithmetic? Or is that comming in a future PR?

Reply:
That's coming in the future